oxapy/__init__.py

```
@@ -8,7 +11,143 @@
+        )
+        handler.on_modified = on_file_changed
+        handler.on_created = on_file_changed
+        handler.on_deleted = on_file_changed
+
+        observer = Observer()
+        observer.schedule(handler, self.__watch_dir, recursive=True)
+        observer.start()
+
+        def spawn_worker() -> subprocess.Popen:
+            """Spawns the child server process with the worker environment flag."""
+            return subprocess.Popen([sys.executable] + sys.argv, env=env)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)